js/custom.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function(){
+
+//user to be able to input text and save it
+$('form').submit(function(event){
+	event.preventDefault();
+	//user text should not be empty
+	var newItem = $('#itemInput').val().trim();
```

Comment:
could be done with a HTML5 `required` attribute in the input tag in the html - might be quicker, but some older browser issues.